controllers/ordersController.js

```diff
@@ -78,17 +78,42 @@ const createOrder = async (req, res) => {
   }
 };
 
-const updateOrder = (req, res) => {
-  return res.status(418).send("Not implemented");
+const updateOrder = async (req, res) => {
+  const updatedOrder = [
```

Comment:
I think it would be better if this were an object, like the create operations? The manager should be the only component that knows how the queries are built.